packages/functional-tests/src/tests/grab-test.ts

```diff
@@ -36,12 +43,10 @@ export default class GrabTest extends Test {
 			}
 		});
 
-		// Load a glTF model
-		this.model = MRE.Actor.CreateFromGltf(this.app.context, {
-			// at the given URL
-			resourceUrl: `${this.baseUrl}/monkey.glb`,
-			// and spawn box colliders around the meshes.
-			colliderType: 'box',
+		// Create an actor
+		this.model = MRE.Actor.CreateFromPrefab(this.app.context, {
```

Comment:
Simply loading a gltf is 2x more expensive now, requiring an additional round trip message. It's also more We need to find a way to move the async part of this operation to the client.

Reply:
If you're only looking at network traffic that's true. I guess I could add an `AssetContainer.loadAndSpawnGltf` method?

---
In reply to: [313511937](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r313511937) [](ancestors = 313511937)

Reply:
Updating this thread: POR is to bring back CreateFromGltf, to preserve this convenience API.

---
In reply to: [313515597](https://github.com/microsoft/mixed-reality-extension-sdk/pull/415#discussion_r313515597) [](ancestors = 313515597,313511937)